.travis.yml

```
@@ -29,7 +29,7 @@ jobs:
         - knitr
         - roxygen2
       r_github_packages:
-        - rformassspectrometry/MsCoreUtils@devel
+        - rformassspectrometry/MsCoreUtils@jomaster
```

```suggestion
        - rformassspectrometry/MsCoreUtils
```
Please change this into `rformassspectrometry/MsCoreUtils` before merging (assuming we merged `group` into `MsCoreUtils`)

---

Jepp. For sure.